src/integrations/dnd5e-integration.ts

```diff
@@ -0,0 +1,315 @@
+/**
+ * D&D 5e Game System Integration for Seasons & Stars
+ *
+ * Provides D&D 5e-specific time sources and compatibility features.
+ * Only loaded when D&D 5e system is detected.
+ */
+
+import { Logger } from '../core/logger';
+
+/**
+ * Simple D&D 5e Integration Manager
+ *
+ * Handles D&D 5e-specific time sources and compatibility features.
+ */
+export class DnD5eIntegration {
+  private static instance: DnD5eIntegration | null = null;
+  private isActive: boolean = false;
+
+  private constructor() {
+    // Private constructor for singleton pattern
+  }
+
+  /**
+   * Initialize D&D 5e integration (called only when D&D 5e system is detected)
+   */
+  static initialize(): DnD5eIntegration {
+    if (DnD5eIntegration.instance) {
+      return DnD5eIntegration.instance;
+    }
+
+    DnD5eIntegration.instance = new DnD5eIntegration();
+    DnD5eIntegration.instance.activate();
+    return DnD5eIntegration.instance;
+  }
+
+  /**
+   * Activate D&D 5e integration
+   */
+  private activate(): void {
+    if (this.isActive) return;
+
+    Logger.info('D&D 5e system detected - enabling enhanced compatibility mode');
+    this.isActive = true;
+
+    // Suggest appropriate calendars for D&D 5e
+    this.suggestCalendars();
+  }
+
+  /**
+   * Suggest appropriate calendars for D&D 5e campaigns
+   */
+  private suggestCalendars(): void {
+    try {
+      const calendarManager = game.seasonsStars?.manager as any;
+      if (!calendarManager) {
+        Logger.debug('Calendar manager not available for D&D 5e calendar suggestions');
+        return;
+      }
+
+      const availableCalendars = calendarManager.getAvailableCalendars();
+      const dnd5eCalendars = availableCalendars.filter(
+        (id: string) =>
+          id.includes('dnd5e') ||
+          id.includes('sword-coast') ||
+          id.includes('forgotten-realms') ||
+          id.includes('faerun')
+      );
+
+      if (dnd5eCalendars.length > 0) {
+        Logger.info(`D&D 5e calendars available: ${dnd5eCalendars.join(', ')}`);
+
+        // Auto-suggest Sword Coast calendar if available
+        const swordCoastCalendar = dnd5eCalendars.find((id: string) => id.includes('sword-coast'));
+        if (swordCoastCalendar) {
+          Logger.info(`Consider using ${swordCoastCalendar} calendar for D&D 5e campaigns`);
```

Comment:
Is this going to pop up on the screen every time the user loads the application or is this only going to the console?